docs/CLI_DESIGN.md

```
@@ -0,0 +1,742 @@
+| `50001` | 服务器内部错误 |
+| `50002` | 任务执行失败 |
+
+---
+
+## AI 交互示例
+
+以下是 AI 与 CLI 交互的典型场景：
+
+### 场景 1: AI 帮助用户创建并执行 Todo
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)